src/components/docs/Excerpt/index.js

```diff
@@ -13,7 +13,9 @@ const Excerpt = ({children, image}) => {
           <img className={styles.image} src={useBaseUrl(image)} />
         </div>
         <div className={styles.text}>
-          {children}
+          <p>
```

Comment:
Breaking up content into paragraphs helps make a page more accessible. `div`s are generic content separators, `p`s let you know that it's text.

Reply:
Good point